subgraph-testnet-cassini-saturn-legends/schema.graphql

```diff
@@ -0,0 +1,120 @@
+type Transfer @entity(immutable: true) {
```

Comment:
Note: subgraph is pretty simple and doesn't handle business logic, ideally it should create entities for easy consumption by the frontend.